R/descriptives.R

```diff
@@ -1239,6 +1289,111 @@ Descriptives <- function(jaspResults, dataset, options) {
   return(p)
 }
 
+
+
+.plotMarginal_Overlay <- function(column, variableName, overlayFactor, splitByName,
```

Comment:
```suggestion
.plotMarginalOverlay <- function(column, variableName, overlayFactor, splitByName,
```
We use camelCase, so no underscores in the names, please. I know that jaspDescriptives does not follow this convention, but it's good to start doing it here.